docs/08-cicd.md

```diff
@@ -1,80 +1,190 @@
 
-# Manual deployment
 
-## iOS
+# Fastlane
 
-## Android
+Fastlane is a task-runner for triggering build relevant tasks. It offers integration with XCode and the Android SDK for building and delivering the app.
 
-`export CREDENTIALS_GIT_REPOSITORY_URL=<secret>`
-`export CREDENTIALS_DIRECTORY_PATH=/tmp/credentials`
-`export CREDENTIALS_KEYSTORE_PASSWORD=<secret>`
-`export CREDENTIALS_KEYSTORE_PATH=/tmp/credentials/<secret>.enc`
-`export KEYSTORE_PATH=/tmp/keystore.jks`
-`fastlane keystore`
+## Setup of Fastlane
 
-`export KEYSTORE_KEY_ALIAS=<secret>`
-`export KEYSTORE_PASSWORD=<secret>`
-`export KEYSTORE_KEY_PASSWORD=<secret>`
-`fastlane build`
+* [Install Ruby >= 2.6.5](https://www.ruby-lang.org/en/documentation/installation/) (The preferred and tested way is using [Ruby Version Manager (RVM)](https://rvm.io/))
+  * If using RVM you have to do: `rvm use ruby-2.6.5`
+* Make sure `ruby --version` reports the correct version
+* Run `bundle install --path vendor/bundle` in the project root, `./android/` and `./ios/`
+* Run `bundle exec fastlane --version`
 
-`adb install app/build/outputs/apk/release/app-release.apk`
-`adb shell am force-stop tuerantuer.app.integreat`
-`adb shell am start -n tuerantuer.app.integreat/.MainActivity`
+*Hint: You can use `export FASTLANE_SKIP_UPDATE_CHECK=true` to skip the changelog output*
 
-## Setup environment variables locally
+## Lanes
 
-# Credentials and services
+There are Android and iOS specific lanes. Lanes for Android live in `./android/fastlane` and for iOS in `./ios/fastlane`.
 
-## deliverino
+### General 
 
-`deliverino` is a GitHub App and can be accessed and installed [here](https://github.com/apps/deliverino). This bot updates the repository when a new release is delivered.
+For an overview of the Fastlane lanes refer to the [auto-generated README](../fastlane/README.md#available-actions). These lanes are responsible for delivering and uploading artifacts, bumping the version using the [deliverino GitHub bot](#deliverino-github) and sending notifications.
 
-Access to the Bot is granted by a Private Key in PEM format. This is used to get an access token for an installation. This access_token allows to write content to the repositories/organisations where it was installed.
+### Android
 
-// PEM is base64 encoded
-// Disable "Include Administrators" in Protected Branches (GitHub App is Admin)
+For an overview of the Fastlane lanes refer to the [auto-generated README](../android/fastlane/README.md#available-actions). These lanes are responsible for setting up the signing keys and building the Android app.
 
+### iOS
+
+For an overview of the Fastlane lanes refer to the [auto-generated README](../ios/fastlane/README.md#available-actions). These lanes are responsible for setting up the certificates and building the iOS app.
+
+# Setup Certificates for iOS on Mac
+
+The setup of certificates is demonstrated in section [Manual builds](06-manual-builds.md#manually-building-for-ios).
```

Comment:
If sections are used in multiple docs, they should probably be moved to a separate file. I don't see an order why some stuff is explained in manual builds and some stuff in cicd.

Reply:
I moved this to a separate file